chevah/compat/tests/normal/test_capabilities.py

```diff
@@ -43,8 +43,17 @@ def runningAsAdministrator(self):
         # 2003 and XP under normal mode.
         if 'win-2003' in self.hostname or 'win-xp' in self.hostname:
             return False
-        else:
-            return True
+
+        return True
+
+    def hasUAC(self):
+        """
+        Return True if Windows version has is UAC capable, False otherwise.
+        """
+        if 'win-2003' in self.hostname or 'win-xp' in self.hostname:
```

Comment:
I think that for this we can do better.
We know that UAC is availalble from Vista and above.

maybe we can use something based on platform.win32_ver()[1]

I get:

* 2003 - > 5.7.3790
* 2008 -> 6.1.7000

so I think that we can use this value